iothub/device/src/Pipeline/AmqpTransportHandler.cs

```diff
@@ -32,7 +31,7 @@ internal class AmqpTransportHandler : TransportHandlerBase
 
         internal IotHubConnectionCredentials _connectionCredentials;
 
-        private static readonly TimeSpan s_twinResponseTimeout = TimeSpan.FromMinutes(60);
+        private static readonly TimeSpan s_twinResponseTimeout = TimeSpan.FromMinutes(2);
```

Comment:
60 minutes was fine for cleaning up memory, but just realized we need to cancel the operation so waiting clients can time out outside of whatever cancellation token they use.

2 minutes seems like long enough for a twin operation to have completed.

Thoughts?

Reply:
This looks good to me!

Reply:
Reverting to 1 hour per Tim and team agreement